tests/custom_language_test.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../db/define_tables'
+require 'sequel'
```

Comment:
I think that in all test you just `require '../app'` you'll get all the gems of the app available.